src/update.c

```diff
@@ -555,6 +555,15 @@ int main_update()
 	run_scripts();
 	grabtime_stop(&times);
 
+	/* Create the state file that will tell swupd it's on a mix on future runs */
+	if (mix_exists && !system_on_mix()) {
+		int fd = open("/usr/share/defaults/swupd/mixed", O_RDWR | O_CREAT, S_IRWXU | S_IRGRP | S_IROTH);
+		if (fd == -1) {
+			fprintf(stderr, "ERROR: Failed to create 'mixed' statefile\n");
```

Comment:
if the file fails to be created, what is the fallout? is it safe to print the error and continue?